### DISCUSSION TITLE
Headers Size

### DISCUSSION THREAD

**Discussion by MartinSchultze** [leonhad/paradoxdriver#100] [General]
Hi leonhad,

if I try to open Paradox Tables with Headers Size more than 2048 DBeaver dosn't show this tables.
Can you fix this Problem?

**Comment by leonhad** [comment]
Hi, yes, I can.

If you can send a test file for me, The fix can be more quickly.

**Comment by MartinSchultze** [comment]
[DuengeBedarf.zip](https://github.com/user-attachments/files/17960145/DuengeBedarf.zip)

**Comment by leonhad** [comment]
Fixed in 1.6.3 version. Please wait for Maven Central to release the new version.
